web/controllers/session_controller.ex

```diff
@@ -0,0 +1,26 @@
+defmodule Chronical.SessionController do
+  use Chronical.Web, :controller
+
+  def new(conn, _) do
+    render conn, "new.html"
+  end
+
+  def create(conn, %{"session" => %{"username" => user, "password" => pass}}) do
+    case Chronical.Auth.login_by_username_and_pass(conn, user, pass, repo: Repo) do
+      {:ok, conn} ->
+        conn
+        |> put_flash(:info, "Welcome back!")
+        |> redirect(to: page_path(conn, :index))
+      {:error, _reason, conn } ->
```

Comment:
There is no whitespace around parentheses/brackets most of the time, but here there is.